stdlib/public/runtime/DynamicCast.cpp

```diff
@@ -863,6 +863,7 @@ tryCastToString(
       takeOnSuccess, mayDeferChecks);
 #endif
   }
+    SWIFT_FALLTHROUGH;
```

Comment:
Nit: maybe move this one line up, inside the braces?

Reply:
Moved in the follow-up commit.